@shared/constants/mercury.ts

```diff
@@ -0,0 +1 @@
+export const INDEXER_URL = "http://localhost:3002/api/v1";
```

Comment:
Should this come from an env var?

Reply:
we do now have the dev instance stood up, and it is currently at `https://freighter-backend-fc55eaf.previews.kube001.services.stellar-ops.com/api/v1`

Reply:
yeah, maybe we should add an `env.development` file

Reply:
@piyalbasu I added an env var for this in 46a4341c09059ee193aa1dd9169dc6a37452f908.

Just noting(mainly for myself later!) that we will need to add this as a secret in github before deployment for the action to use it.

Reply:
jus deployed an update to the backend, so the new remote instance is at https://freighter-backend-88d8d43.previews.kube001.services.stellar-ops.com/